extensions/podman/src/podman-install.ts

```diff
@@ -249,6 +311,17 @@ export class PodmanInstall {
         await extensionApi.window.showWarningMessage('Podman update has been canceled.', 'OK');
         return;
       }
+
+      // podman v4 -> v5 migration: ask to wipe all data before doing the update
+      const wipeAllDataCompleted = await this.wipeAllDataBeforeUpdatingToV5(installedPodman, updateInfo);
+      if (!wipeAllDataCompleted) {
+        await extensionApi.window.showWarningMessage(
+          'Podman update has been canceled. You can do some backups before restarting the update',
```

Comment:
Would it be preferred to use a "it is recommended" instead of "you can do"? Something like `It is recommended to perform backups before resuming the update procedure.`